src/clients.html

```diff
@@ -11,11 +11,58 @@
   </head>
   <body>
 
-    <!-- TO-DO: update menu when designs are finalized. Start Menu -->
+    <!-- Start Menu -->
+    <nav>
+      <div class="studio-carbon-negative-button-container fixed flex col-6 items-center transition-shorter z-2000">
+        <button href="index.html" id="studio-carbon-negative-button" class="button--style-nav text-menu-md serif studio-carbon-negative-button studio-carbon-negative-button--inactive transition-shorter">
+          <span class="italic mr_25">Studio</span><span>Carbon Negative</span>
+        </button>
+      </div>
 
-    <!-- End Menu -->
+      <div class="nav-menu-button-container fixed z-overlay flex items-center col-12 xl:col-6 xl:off-6 justify-between">
+        <div class="table-of-contents opacity-0 transition-shorter col-6 text-menu-md serif">
+          Table of Contents
+        </div>
+        <button id="nav-menu-button" class="col-6 flex justify-end button--style-nav text-left italic text-menu-md serif transition-shorter">
+          Menu
+        </button>
+        <button id="nav-close-button" class="button--style-nav text-left flex justify-end italic text-menu-md serif none opacity-0 events-none transition-shorter">
+          Close
+        </button>
+      </div>
 
-    <!-- Start Page Content -->
+      <div id="menu" class="fixed w100 menu menu--inactive">
+        <div class="inner-menu mxauto w100 h100">
+          <div class="inner-menu-inner-container">
+            <div class="text-menu-lg serif xl:col-6 xl:off-6 links mb2_5 xl:mb2">
+              <div>
+                <button class="button--style-nav text-left menu-link opacity-0"><a href="index.html">Studio Carbon Negative</a></button>
+              </div>
+              <div>
+                <button class="button--style-nav text-left menu-link opacity-0"><a href="emissions.html">Calculating Emissions</a</button>
+              </div>
+              <div>
+                <button class="button--style-nav text-left menu-link opacity-0"><a href="offsetting.html">Buying Offsets</a</button>
+              </div>
+              <div>
+                <button class="button--style-nav text-left menu-link opacity-0"><a href="clients.html">Helping Clients</a</button>
+              </div>
+              <div>
+                <button class="button--style-nav text-left menu-link opacity-0"><a href="resources.html">Resources</a></button>
```

Comment:
If you are using a tag you don't have to wrap that with button tag.
We need aria-label for buttons